sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroSource.java

```diff
@@ -393,7 +393,7 @@ private static InputStream decodeAsInputStream(byte[] data, String codec) throws
       ByteArrayInputStream byteStream = new ByteArrayInputStream(data);
       switch (codec) {
         case DataFileConstants.SNAPPY_CODEC:
-          return new SnappyCompressorInputStream(byteStream);
+          return new SnappyCompressorInputStream(byteStream, 1 << 16 /* Avro uses 64KB blocks */);
```

Comment:
I assume this was done because Avro's `Codec` does not support the interface we need: https://avro.apache.org/docs/1.8.1/api/java/org/apache/avro/file/Codec.html